### PR TITLE
Fix clickhouse tests

### PR DESCRIPTION
**Pull request by Abacn**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Abacn** [comment]
R: @shunping

File: sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/ClickHouseIOTest.java [hunk 1]
```diff
@@ -391,22 +391,22 @@ public void testArrayOfPrimitiveTypes() throws Exception {
...
-      assertEquals("[7,8]", rs.getString("f7"));
-      assertEquals("['eight','nine']", rs.getString("f8"));
-      assertEquals("[9,10]", rs.getString("f9"));
-      assertEquals("[10,11]", rs.getString("f10"));
-      assertEquals("[11,12]", rs.getString("f11"));
-      assertEquals("[12,13]", rs.getString("f12"));
-      assertEquals("['abc','cde']", rs.getString("f13"));
-      assertEquals("['cde','abc']", rs.getString("f14"));
-      assertEquals("[true,false]", rs.getString("f15"));
+      assertEquals("[2030-10-01, 2031-10-01]", rs.getString("f0"));
```
Comment (shunping) [inline sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/ClickHouseIOTest.java:394]: Looks like all the string fields no longer have single quotes around them. Is that the breaking change introduced by the clickhouse-server version upgrade?

File: sdks/java/io/clickhouse/build.gradle [hunk 2]
```diff
@@ -66,7 +66,6 @@ dependencies {
   testImplementation library.java.testcontainers_clickhouse
   testRuntimeOnly library.java.slf4j_jdk14
   testRuntimeOnly project(path: ":runners:direct-java", configuration: "shadow")
-  testImplementation project(path: ":sdks:java:extensions:avro", configuration: "testRuntimeMigration")
```
Comment (shunping) [inline sdks/java/io/clickhouse/build.gradle:69]: unused dependency?

Comment (Abacn) [reply sdks/java/io/clickhouse/build.gradle:69]: yes

File: sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/ClickHouseIOTest.java (hunk 1 above)
Comment (Abacn) [reply sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/ClickHouseIOTest.java:394]: After I resolve the connection issue I already see these breaking changes without bump clickhouse/clickhouse-server and zookeeper docker version.

That said the different string is due to testcontainer 1.16 -> 1.19 version change, however which exact component of test container caused this is not known

**Review by shunping — APPROVED** [review]
LGTM

**Pull request merged**